e2e/e2e_test.go

```diff
@@ -1094,1316 +1129,9 @@ func Test_Hedera_EVM_BurnERC721_Transfer(t *testing.T) {
 	}
 
 	// 10. Wait for validators to update DB state after Scheduled TX is mined
-	time.Sleep(20 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.